velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -403,7 +463,8 @@ HiveDataSink::HiveDataSink(
           hiveConfig,
           bucketCount,
           std::move(bucketFunction),
-          getPartitionChannels(insertTableHandle),
+          insertTableHandle->partitionChannels(),
+          insertTableHandle->nonPartitionChannels(),
           nullptr) {}
```

Comment:
You'll need to change the nullptr here to 
```suggestion
          createPartitionIdGenerator(
              inputType,
              insertTableHandle,
              hiveConfig,
              connectorQueryCtx)) {}
```

You'll also need to remove the std::move from `std::move(inputType)` above to avoid a use-after-move issue.